sickbeard/name_parser/parser.py

```diff
@@ -346,19 +347,21 @@ def _combine_results(self, first, second, attr):
         b = getattr(second, attr)
 
         # if a is good use it
-        if a != None or (type(a) == list and len(a)):
+        if a != None or (isinstance(a, list) and a):
```

Comment:
Yes that is kinda silly, but I will leave that as is for now as this PR is explicitly for linting fixes.

Code changes and refactoring should be kept separate.